src/bindings/c/include/openvino/c/ov_tensor.h

```diff
@@ -24,6 +24,38 @@ typedef struct ov_tensor ov_tensor_t;
  * @{
  */
 
+/**
+ * @struct ov_tensor_list_t
+ * @brief A list of tensors
+ */
+typedef struct ov_tensor_list ov_tensor_list_t;
```

Comment:
Why do we need to have tensor list hidden structure?
Can we use C array instead?

Reply:
ov_tensor is opaque handle of ov::Tensor, so if apply ov_tensor into C array, the array must be pre-allocated and the size is fixed. Will it make sense?